infrastructure/comms/shared_struct/shared_struct_publisher.inl

```diff
@@ -0,0 +1,43 @@
+
+namespace jet {
+
+SharedStructPublisher::SharedStructPublisher(const std::string& object_name) : Publisher(object_name) {
+  int o_flags = O_RDWR | O_CREAT;
+  off_t length = 921650 + 1024;
```

Comment:
Consider making these named constants (and making them `const`)

Reply:
:+1: